tests/Phinx/Config/ConfigDefaultEnvironmentTest.php

```diff
@@ -37,16 +38,15 @@ public function testConfigReplacesTokensWithEnvVariables()
         $this->assertEquals('1234', $env['port']);
     }
 
-    /**
-     * @expectedException \RuntimeException
-     * @expectedExceptionMessage The environment configuration (read from $PHINX_ENVIRONMENT) for 'conf-test' is missing
-     */
     public function testGetDefaultEnvironmentOverridenByEnvButNotSet()
     {
         // set dummy
         $dummyEnv = 'conf-test';
         putenv('PHINX_ENVIRONMENT=' . $dummyEnv);
 
+        $this->expectException(RuntimeException::class);
+        $this->expectExceptionMessage('The environment configuration (read from $PHINX_ENVIRONMENT) for \'conf-test\' is missing');
+
         try {
```

Comment:
This `try-catch` is better as `try-finally`, because in case exception is not thrown, environment wouldn't reset, there is also no need to manually rethrow exception.